rust/Cargo.lock

```diff
@@ -1,28 +1,22 @@
 # This file is automatically @generated by Cargo.
```

Comment:
I'm really surprised this is checked in; not your fault for this folder. I'm tempted to remove it, add it to git ignore, and rebuild.